Algorithm.Framework/Portfolio/MinimumVariancePortfolioOptimizer.py

```diff
@@ -63,7 +63,7 @@ def Optimize(self, historicalReturns, expectedReturns = None, covariance = None)
                        constraints = constraints,                                 # Constraints definition
                        method='trust-constr')   # Optimization method:  trust-region algorithm for constrained optimization
 
-        return opt['x'] if opt['success'] else x0
+        return opt['x'] / np.sum(abs(opt['x'])) if opt['success'] else x0
```

Comment:
This solution will break the budget constraint:  Σw = 1 will become Σw <= 1. Perhaps it's possible to add another constraint where Σ|w| <= 1, and the minimize function would find a solution. I am not sure how this is done in C# using this library, but for python we should be able to use something like:

```python
        constraints = [
            {'type': 'eq', 'fun': lambda weights: self.get_budget_constraint(weights)},
            {'type': 'eq', 'fun': lambda weights: self.get_absolute_budget_constraint(weights)},   # new constraint
            {'type': 'eq', 'fun': lambda weights: self.get_target_constraint(weights, expectedReturns)}]

    def get_absolute_budget_constraint(self, weights):
        '''Defines a budget constraint: the sum of the absolute weights equals unity'''
        return np.sum(np.abs(weights)) - 1
```